docs/Installation.md

```diff
@@ -110,19 +106,19 @@ If you get any warnings or recommendations, fix them before moving on. Don't cli
 yet though, since you'll get errors if you do...
 
 
-4) Generating your website
+4) Generating your Website
 --------------------------
 
 First, you should generate a Symfony bundle for your website.
 
-    app/console kuma:generate:bundle
+    php app/console kuma:generate:bundle
```

Comment:
Windows doesn't support sebang and thus requires the `php` prefix. Unix users either know they can remove this prefix or don't bother (as it doesn't cause trouble).